include/clipboard.h

```diff
@@ -0,0 +1,37 @@
+/*
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *
+ *  Copyright (C) 2023-2024  The DOSBox Staging Team
```

Comment:
2024-2024

Reply:
Changed - it is actually based on the code I wrote the last year, that's why it was 2023-2024.